packages/nimble-components/src/icon-base/template.ts

```diff
@@ -3,6 +3,6 @@ import type { Icon } from '.';
 
 export const template = html<Icon>`
     <template>
```

Comment:
can remove the wrapping template if it has no bindings on it.
think we should also give it the no whitespace in the template treatment and a comment stating that because it is used inline.